src/public/javascripts/batteries.js

```diff
@@ -1,225 +1,225 @@
-  cell.setVoltage(voltage);
-}
-
-/**
- * Updates battery cell with voltage and renders it on the table
- * @param {Number} batteryIndex Index of battery to update
- * @param {Number} voltage Voltage to put into battery
- */
-function updateBatteryAndRender(batteryIndex, voltage) {
-  updateBattery(batteryIndex, voltage);
-  updateCell(batteryIndex);
-}
-
-/**
- * Updates the label at bottom of screen with temperatures
- * @param {Number} min Minimum temperature of battery pack
- * @param {Number} avg Average temperature of battery pack
- * @param {Number} max Maximum termperature of battery pack
- */
-function updateTemps(min, avg, max) {
-  document.getElementById('temps').innerHTML = `Min Cell Temp: ${min} C   Avg Cell Temp: ${avg} C   Max Cell Temp: ${max} C`;
-}
-
-/**
- * Checks if a cell's voltage is within the nominal values
- */
-function checkMinMax() {
-  let cell;
-  let tableCell;
-  let voltNum;
-  for (let i = 0; i < NUM_CELLS; i++) {
-    cell = batteries[i];
-    // console.log(cell);
-    tableCell = document.getElementById(`${cell.row}${cell.col}volt`).innerHTML;
-    voltNum = Number(tableCell.substr(9));
-    // console.log(`${i} ${tableCell}`);
-    if (voltNum <= MIN_VOLTAGE || voltNum >= MAX_VOLTAGE) {
-      document.getElementById(`${cell.row}${cell.col}cell`).style.backgroundColor = 'red';
-    } else {
-      document.getElementById(`${cell.row}${cell.col}cell`).style.backgroundColor = 'green';
-    }
-  }
-}
-// Check on given interval
-setInterval(checkMinMax, 1000);
-
-// On data in update and render voltages/ temps
-client.on('dataIn', (input) => {
-  console.log(input);
-  if (input.battery.cells) {
-    let cells = [];
-    for (let i = 0; i < input.battery.cells.length; i++) {
-      cells[i] = input.battery.cells[i];
-    }
-    for (let i = 0; i < cells.length; i++) {
-      console.log(`index ${i} volts ${cells[i]}`);
-      updateBatteryAndRender(i, cells[i]);
-    }
-    updateTemps(input.battery.minCellTemp, input.battery.avgCellTemp, input.battery.maxCellTemp);
-  }
-});
+/**
+ * Batteries Modules
+ * @module Batteries
+ */
+const client = require('./public/javascripts/communication').recievedEmitter;
+
+/** @constant {Number} - Width of battery table */
+const TABLE_WIDTH = 8;
+
+/** @constant {Number} - Height of battery table */
+const TABLE_HEIGHT = 9;
+
+/** @constant {Number} - Number of Cells to be in Battery Index */
+const NUM_CELLS = TABLE_WIDTH * TABLE_HEIGHT;
+
+/** @constant {Number} - Maximum nominal voltage */
+const MAX_VOLTAGE = 4.2;
+
+/** @constant {Number} - Minimum nominal voltage */
+const MIN_VOLTAGE = 3.1;
+
+/** @constant {HTMLElement} - HTML Element of Table */
+const TABLE = document.getElementById('batteryTable');
+
+let batteries = [];
+/**
+ * Represents a single battery cell
+ * @constructor
+ * @param {String} name - name of battery cell
+ * @param {Number} row - row it belongs to
+ * @param {Number} col - column it belongs to
+ */
+function Battery(name, row, col) {

```

Comment:
A little wonky that you decided to ditch the syntactic sugar of using 'class' just for this one

Reply:
This is from California I'm not sure how it got in this pull request, probably best to make it conform with the rest of the codebase however